.gitignore

```diff
@@ -2,3 +2,5 @@
 composer.lock
 node_modules
 vendor
+
+.idea/
```

Comment:
Ugh, should be in my local exclude sorry. kind of switching back to sublime anyway :)